File: Python/tests/test_igt_pvl_decay.py
import pytest

from hbayesdm.models import igt_pvl_decay


def test_igt_pvl_decay():
    _ = igt_pvl_decay(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_igt_pvl_decay.R
context("Test igt_pvl_decay")
library(hBayesDM)

test_that("Test igt_pvl_decay", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(igt_pvl_decay(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
